packages/engine/Source/Core/Rectangle.js

```diff
@@ -542,7 +542,7 @@ Rectangle.prototype.equalsEpsilon = function (other, epsilon) {
  * @exception {DeveloperError} <code>east</code> must be in the interval [<code>-Pi</code>, <code>Pi</code>].
  * @exception {DeveloperError} <code>west</code> must be in the interval [<code>-Pi</code>, <code>Pi</code>].
  */
-Rectangle.validate = function (rectangle) {
+Rectangle._validate = function (rectangle) {
```

Comment:
As we discussed last night we should go through the deprecation process for this function. You can read about it in our [Coding Guide](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#deprecation-and-breaking-changes). Basically rename this function but also add the `Rectangle.validate` function back which just calls through to this one and has the `@deprecated` tag and `deprecatedWarning`.